### DISCUSSION TITLE
Why the first term of my loss function is zero?

### DISCUSSION THREAD

**Discussion by thevelvetunderground** [lululxvi/deepxde#1157] [Q&A]
<img width="893" alt="image" src="https://user-images.githubusercontent.com/91045087/220555483-afea7d7c-b3a5-4d21-b09c-1c40d433547d.png">
<img width="587" alt="image" src="https://user-images.githubusercontent.com/91045087/220555564-9f7e3f74-19ae-4ab8-b5ce-fba642181ee1.png">
<img width="365" alt="image" src="https://user-images.githubusercontent.com/91045087/220555653-2c86ffa7-f1a2-485e-90f4-5a5749870cc6.png">
I use the same PDE in other progam too, but it works.By the way, whats the order of the loss? The issue:"What is the output of DeepXDE? How can I visualize the results?" seems closed.

**Comment by lululxvi** [comment]
Code looks OK. Not sure what happened here.

Here is FAQ https://deepxde.readthedocs.io/en/latest/user/faq.html. I don't understand what is "closed".
